pkg/pipeline/ImageTaggingService.go

```diff
@@ -50,27 +51,26 @@ type ImageTaggingService interface {
 	GetProdEnvByCdPipelineId(pipelineId int) (bool, error)
 	// ValidateImageTaggingRequest validates the requested payload
 	ValidateImageTaggingRequest(imageTaggingRequest *types.ImageTaggingRequestDTO, appId, artifactId int) (bool, error)
-	GetTagNamesByArtifactId(artifactId int) ([]string, error)
 	// GetTagsDataMapByAppId this will fetch a map of artifact vs []tags for given appId
 	GetTagsDataMapByAppId(appId int) (map[int][]*repository.ImageTag, error)
 	// GetImageCommentsDataMapByArtifactIds this will fetch a map of artifact vs imageComment for given artifactIds
 	GetImageCommentsDataMapByArtifactIds(artifactIds []int) (map[int]*repository.ImageComment, error)
-	// GetUniqueTagsByAppId gets all the unique tag names for the given appId
-	GetUniqueTagsByAppId(appId int) ([]string, error)
 	GetImageTaggingServiceConfig() ImageTaggingServiceConfig
 	FindProdEnvExists(externalCi bool, pipelineIds []int) (bool, error)
 }
 
 type ImageTaggingServiceImpl struct {
```

Comment:
can move this service to new path , i.e. /build/artifacts/imageTagging

Reply:
updated 👍